### PR TITLE
Prl 5844

### PR DESCRIPTION
**Pull request by sahitya-desireddy**
https://tools.hmcts.net/jira/browse/PRL-5844

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/prl/models/email/EmailTemplateNames.java [hunk 1]
```diff
@@ -57,7 +57,9 @@ public enum EmailTemplateNames {
     FM5_REMINDER_APPLICANT_RESPONDENT,
     RESPONDENT_RESPONDED_CAFCASS,
     RESPONDENT_ALLEGATIONS_OF_HARM_CAFCASS,
-    RESPONDENT_RESPONDED_ALLEGATIONS_OF_HARM_CAFCASS
+    C7_NOTIFICATION_APPLICANT, C1A_NOTIFICATION_APPLICANT, RESPONDENT_RESPONDED_ALLEGATIONS_OF_HARM_CAFCASS,
```
Comment (yogendraU) [inline src/main/java/uk/gov/hmcts/reform/prl/models/email/EmailTemplateNames.java:60]: can we please follow same standard as having one template name per line

Comment (sahitya-desireddy) [reply src/main/java/uk/gov/hmcts/reform/prl/models/email/EmailTemplateNames.java]: Done @yogendraU Thank you !! Request please verify

File: src/main/java/uk/gov/hmcts/reform/prl/services/reviewdocument/ReviewDocumentService.java [hunk 2]
```diff
@@ -324,6 +358,30 @@ private boolean processReviewDocument(CaseData caseData,
     private void sendNotifications(CaseData caseData, Element<QuarantineLegalDoc> quarantineLegalDocElementOptional,
                                    String quarantineDocsListToBeModified) {
         sendNotificationToCafCass(caseData,quarantineLegalDocElementOptional,quarantineDocsListToBeModified);
+        if (C100_CASE_TYPE.equalsIgnoreCase(CaseUtils.getCaseTypeOfApplication(caseData))) {
```
[the remark this answers is not in the document]
Comment (sahitya-desireddy) [reply src/main/java/uk/gov/hmcts/reform/prl/services/reviewdocument/ReviewDocumentService.java]: Done @yogendraU Thank you !! Request to please verify

File: src/main/java/uk/gov/hmcts/reform/prl/services/reviewdocument/ReviewDocumentService.java [hunk 3]
```diff
@@ -365,6 +423,106 @@ private void sendNotificationToCafCass(CaseData caseData, Element<QuarantineLega
...
     }
 
+    private void sendNotificationToApplicant(String authorisation, CaseData caseData,
+                                             EmailTemplateNames emailTemplate,
+                                             SendgridEmailTemplateNames partyEmailTemplate,
+                                             SendgridEmailTemplateNames solicitorEmailTemplate) {
+        if (YesNoNotSure.no.equals(caseData.getReviewDocuments().getReviewDecisionYesOrNo())) {
+            List<PartyDetails> applicantToNotify = nullSafeCollection(caseData.getApplicants()).stream()
+                .map(Element::getValue)
+                .filter(applicant -> CaseUtils.hasLegalRepresentation(applicant) || Yes.equals(applicant.getCanYouProvideEmailAddress()))
```
[the remark this answers is not in the document]
Comment (sahitya-desireddy) [reply src/main/java/uk/gov/hmcts/reform/prl/services/reviewdocument/ReviewDocumentService.java]: Removed this extra legal representation and iterating overall the applicants and sending email according to the conditions please verify @yogendraU Thank you !!

**Review by sahaSonali — APPROVED** [review]

**Pull request merged**